src/ape/managers/converters.py

```diff
@@ -38,6 +45,28 @@ def convert(self, value: str) -> bytes:
         return HexBytes(value)
 
 
+class HexIntConverter(ConverterAPI):
+    """
+    Convert hex values to integers.
+    """
+
+    def is_convertible(self, value: Any) -> bool:
+        return (
+            isinstance(value, int)
```

Comment:
Should skip this value tbh, as it's just pass-through since if the value is already the target type, it skips conversion

Reply:
https://github.com/ApeWorX/ape/pull/1248